retis/src/module/nft/bpf/nft.bpf.c

```diff
@@ -17,28 +18,30 @@
 #define retis_get_nft_type(ctx, cfg)		\
 	RETIS_HOOK_GET(ctx, cfg->offsets, nft_type, enum nft_trace_types)
 
-/* Nft hook configuration.
+/** 
+ * Nft hook configuration.
+ * 
```

Comment:
There are extra spaces at EOL in this file.

Reply:
this was needed to instruct bindgen with an "annotating" comment/doc.
Must be converted back to C style.

Reply:
ok, the trailing whitespaces need to be removed. The comment has to stay instead. Plain C-style doesn't make bindgen pick this annotation up.